functions.php

```diff
@@ -66,15 +71,19 @@ function judge_familiar_page_navigation_box( $page ) {
 
 		$prev = get_post_meta( $page->ID, '_previous_page', true );
 		$next = get_post_meta( $page->ID, '_next_page', true );
+		$prev_title = get_post_meta( $page->ID, '_previous_page_title', true );
+		$next_title = get_post_meta( $page->ID, '_next_page_title', true );
 ?>
 		<div class="page_navigation_input">
 			<label for="page_navigation_input_prev">Previous Page</label>
-			<input id="page_navigation_input_prev" name="page_navigation_input_prev" type="text" value="<?= $prev ?>" />
+			<input id="page_navigation_input_prev" name="page_navigation_input_prev" type="number" value="<?= $prev ?>" placeholder="Page ID"/><!--
+			--><input id="page_navigation_input_prev_title" name="page_navigation_input_prev_title" type="text" value="<?= $prev_title ?>" placeholder="Page Title" />
```

Comment:
No problem if you don't like the comment, I removed it.
I still don't think that we should fix HTML shortcomings with CSS styles, when they can be fixed just as easily within the HTML.